xonsh/commands_cache.py

```diff
@@ -0,0 +1,142 @@
+# -*- coding: utf-8 -*-
+import builtins
+import os
+from collections import abc as abc
```

Comment:
This should be `import collections.abc as abc` to allow this to be a lazy import